### PR TITLE
[IMP][9.0] contract: Add report + send by mail

### PR DESCRIPTION
**Pull request by cubells**
This PR adds the following features to contract module:

- a report to print contract data.
- a button to send by email contract report.

When send email, information is saved in contract chatter.

cc @Tecnativa

### CODE REVIEW

File: contract/__openerp__.py [hunk 1]
```diff
@@ -15,7 +15,10 @@
     'depends': ['base', 'account', 'analytic'],
     'data': [
         'security/ir.model.access.csv',
+        'report/report_contractorder.xml',
```
Comment (pedrobaeza) [inline contract/__openerp__.py]: Better just call it `report_contract.xml`

**Review by pedrobaeza — APPROVED** [review]
@rafaelbn @chienandalu please review

**Review by chienandalu — CHANGES_REQUESTED** [review]
The contract report blocks appear with an ugly border sorrounding them:
image

**Review by carlosdauden — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@cubells please make the style changes. Use invoice report as template.

**Comment by cubells** [comment]
done.

**Review by chienandalu — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@cubells please make the same changes in 10.0

**Pull request merged**